cmd/agent/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
The CentOS image isn't that big (192.5 MB), which is shared by all images. But if you have a specific image to suggest, I could try that out and see if it works.